VimeoNetworking/VimeoNetworking/ResponseCache.swift

```diff
@@ -90,19 +99,19 @@ final internal class ResponseCache
             self.cache.setObject(responseDictionary, forKey: key)
         }
         
-        private func responseDictionaryForKey(key: String) -> VimeoClient.ResponseDictionary?
+        private func responseDictionary(forKey key: String, completion: ResponseDictionaryClosure)
```

Comment:
Can you make the closure function optional and introduce a return val function that's also optional. Or do something to enforce one or the other. Or just implement both?